src/utils/bundleFeedUtils.ts

```diff
@@ -50,16 +64,34 @@ export namespace bundleFeedUtils {
         }
     }
 
-    export async function getRelease(context: IActionContext, bundleMetadata: IBundleMetadata | undefined, templateVersion: string): Promise<ITemplatesRelease> {
+    export async function getRelease(context: IActionContext, bundleMetadata: IBundleMetadata | undefined, templateVersion: string): Promise<ITemplatesReleaseV1> {
         const feed: IBundleFeed = await getBundleFeed(context, bundleMetadata);
         return feed.templates.v1[templateVersion];
     }
 
-    export function isBundleTemplate(template: IFunctionTemplate | IBindingTemplate): boolean {
+    export async function getReleaseV2(context: IActionContext, bundleMetadata: IBundleMetadata | undefined, templateVersion: string): Promise<ITemplatesReleaseV2> {
+        const feed: IBundleFeed = await getBundleFeed(context, bundleMetadata);
+        return feed.templates.v2[templateVersion];
+    }
+
+    export function isBundleTemplate(template: IFunctionTemplate | IBindingTemplate | FunctionV2Template): boolean {
         const bundleTemplateTypes: string[] = ['durable', 'signalr'];
+        // I don't think v2 templates are ever a bundle template? Will have to verify this
```

Comment:
What's the verdict

Reply:
gif caught my attention 😅

Yeah they should be bundle templates. For example, we should have a different template for a Cosmos DB trigger depending if it's a newer bundle or old bundle. Here are examples of how the code is different:
- New bundle: https://github.com/Azure/azure-functions-nodejs-e2e-tests/blob/main/app/v4/src/functions/cosmosDBTrigger1.ts
- Old bundle: https://github.com/Azure/azure-functions-nodejs-e2e-tests/blob/main/app/v4-oldBundle/src/functions/cosmosDBTrigger1.ts

If this isn't the case, we can have a chat with Naren and I'm sure he'll sort us out.

Reply:
Sorry, what I meant is, I assume that they are all "true" and since it wouldn't be included in the v1 runtime, we don't ever call it in the first place.

So nah, we don't need this.